src/content/docs/infrastructure/infrastructure-ui-pages/hosts-new-view.mdx

```diff
@@ -0,0 +1,173 @@
+
+import infrastructureHostsNewSummary from 'images/infrastructure_screenshot-full_hosts-new-summary.png'
+
+import infrastructureSavedViewsBttn from 'images/infrastructure_icon_saved-views-bttn.png'
+
+import infrastructureExportCSV from 'images/infrastructure_icon_export-csv.png'
+
+import infrastructureHostNewSystem from 'images/infrastructure_screenshot-full_host-new-system.png'
+
+import infrastructureHostNewNetwork from 'images/infrastructure_screenshot-full_host-new-network.png'
+
+import infrastructureHostNewProcesses from 'images/infrastructure_screenshot-full_host-new-processes.png'
+
+import infrastructureHostNewStorage from 'images/infrastructure_screenshot-full_host-new-storage.png'
+
+import infrastructureHostNewContainers from 'images/infrastructure_screenshot-full_host-new-containers.png'
+
+
+<Callout title="PREVIEW FEATURE">
+    This feature is currently in preview. If you want to try it out, contact your New Relic representative.
+</Callout>
+
+Use the **Hosts** page to better understand important performance metrics, like resource usage, network and processes performance, and data about your containers. You can filter your hosts by attributes and compare their performance with inventory change events to help determine root causes. You can also set alert notifications to help prevent problems.
+
+## Access the new infrastructure UI [#access-new-ui]
+
+To view your hosts information, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > Hosts (New)**.
+
+<img
+  title="Hosts (New) Summary page"
+  alt="Hosts (New) Summary page"
+  src={infrastructureHostsNewSummary}
+/>
+
+<figcaption>
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > Hosts (New)**
+</figcaption>
+
+## Filters and saved views [#filter]
+
+Use the filter at the top of the page to find your host by using the suggested filters, or by typing its name or tags attached to it.
+
+If you have any saved views for Infrastructure, you can access them through the <img src={infrastructureSavedViewsBttn} class="inline" style={{height: "30px", width: "95px"}} alt="Saved views" title="Saved views" /> button.
+
+## Tabs
+
+These are the available tabs in the **Hosts** page:
+
+<CollapserGroup>
+    <Collapser
+    id="summary"
+    title="Summary"
+    >
+    The **Summary** tab gives you an overview of your hosts' performance.
+
+    You can see the total count of your hosts, applications, events, and alerts active in your system.
+
+    In the **Summary** table, you can see a list of your hosts.
+    </Collapser>
+
+    <Collapser
+    id="system"
+    title="System"
+    >
+    The **System** tab gives you an overview of your system. 
+
+    In the **System** table, you can filter your hosts by metric name. With the <img src={infrastructureExportCSV} class="inline" style={{height: "30px", width: "30px"}} alt="Export CSV" title="Export CSV" /> button, you can export this table as a CSV file.
+
+    <img
```

Comment:
@x8a it looks like some of your screenshots may have production names. Do you need to blur these if they're not taken from Demotron?

Reply:
I thought it was ok to display NR admin, but I can blur it out.

Reply:
@x8a NR Admin has production information we should obfuscate with a blur.